src/Aspire.Hosting.Azure.EventHubs/api/Aspire.Hosting.Azure.EventHubs.cs

```diff
@@ -12,15 +12,25 @@ public static partial class AzureEventHubsExtensions
     {
         public static ApplicationModel.IResourceBuilder<Azure.AzureEventHubsResource> AddAzureEventHubs(this IDistributedApplicationBuilder builder, string name) { throw null; }
 
+        [System.Obsolete("This method is obsolete and will be removed in a future version. Use WithHub instead to add an Azure Event Hub.")]
         public static ApplicationModel.IResourceBuilder<Azure.AzureEventHubsResource> AddEventHub(this ApplicationModel.IResourceBuilder<Azure.AzureEventHubsResource> builder, string name) { throw null; }
 
+        public static ApplicationModel.IResourceBuilder<Azure.AzureEventHubsEmulatorResource> ConfigureEmulator(this ApplicationModel.IResourceBuilder<Azure.AzureEventHubsEmulatorResource> builder, System.Action<System.Text.Json.Nodes.JsonNode> configJson) { throw null; }
```

Comment:
Should this be `WithConfiguration`? 

```C#
builder.AddEventHubs("hubs")
           .RunAsEmulator(e => e.WithConfiguration(... ));
```

Reply:
cc @sebastienros

Reply:
Just to clarify:
- based on the model we generate a JSON document
- there is a `WithConfigurationFile` to provide an existing JSON document. 
- `ConfigureEmulator` can alter the JSON document that was generated by the mode (not from a custom file)

OK to change the name, it's new, in Event Hubs and Service Bus.